cypress/support/gui-commands.js

```diff
@@ -0,0 +1,3 @@
+Cypress.Commands.add('getByClassThatStartsWith', (classPart) => {
```

Comment:
```suggestion
Cypress.Commands.add('getByClassThatStartsWith', classPart => {
```

_Arrow functions_ que recebem só um argumento podem omitir os parênteses.

Fica a sugestão, não precisa aceitar se preferir o uso dos parênteses.

Eu sou econômico, e portanto, alguns caracteres a menos ajudam.

Reply:
removido os parenteses conforme sugerido.